cohortextractor/patients.py

```diff
@@ -1528,8 +1532,12 @@ def with_complete_gp_consultation_history_between(
     return_expectations=None,
 ):
     """
+    This captures patients who are registered with one practice between the start date and end date
+    and that this practice used SystmOne continuously between those dates.
```

Comment:
Consistency is important! As we're comparing two functions here, and not all the functions in this module, how about:

> All patients registered with the same practice through the given period, when the practice used SystemOne through the given period.

Reply:
Are we sure about it only identifying SystmOne (note spelling @iaindillingham :D)?

The function name implies something different. What does it do when you run it against an EMIS backend?

Reply:
Oh, *that* SystmOne! 🤦🏻‍♂️